helm-charts/helm-api/ibm_cloud_pak/manifest.yaml

```diff
@@ -1,111 +1,142 @@
 # Licensed Materials - Property of IBM
 # (C) Copyright IBM Corporation 2016, 2019. All Rights Reserved.
 # US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
-
-charts:	
-  - archive: file:helm-api-3.5.0.tgz
-    repository-keys:
-    - helmapi.image.repository
-    - rudder.image.repository
-    - auditService.image.repository
 images:	
 - image: icp-helm-api:3.3.0
-  references:	
-  - repository : icp-helm-api-amd64:3.3.0
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-amd64/icp-helm-api:3.3.0
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: amd64
-  - repository : icp-helm-api-ppc64le:3.3.0
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-ppc64le/icp-helm-api:3.3.0
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: ppc64le
-  - repository : icp-helm-api-s390x:3.3.0
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-s390x/icp-helm-api:3.3.0
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: s390x
-
+  references:
+  - repository: icp-helm-api:3.3.0
+    pull-repository: quay.io/opencloudio/icp-helm-api:3.3.0
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-helm-api:3.3.0
+    pull-repository: quay.io/opencloudio/icp-helm-api:3.3.0
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-helm-api:3.3.0
+    pull-repository: quay.io/opencloudio/icp-helm-api:3.3.0
+    platform:
+      os: linux
+      arch: s390x
+- image: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+  references:
+  - repository: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    pull-repository: quay.io/opencloudio/icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    pull-repository: quay.io/opencloudio/icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    pull-repository: quay.io/opencloudio/icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
```

Comment:
icp-helm-api is at 3.4.1 and digest is `e73ac971497212eb3b8904918aa975136c8e8e8c47a520273e51e65fd9bff214` now